pkg/image/resolve_test.go

```diff
@@ -193,83 +192,6 @@ func TestResolveWithCluster(t *testing.T) {
 	}
 }
 
-func TestGetImages(t *testing.T) {
-	// Setup a httpserver
-	server := setupTestServer()
-	defer server.Close()
-
-	// Mock the parseRepoName function to return the expected repoName
-	originalParseRepoName := parseRepoName
-	parseRepoName = func(url string) (string, error) {
-		return "rancher/ui-plugin-catalog", nil
-	}
-	defer func() {
-		parseRepoName = originalParseRepoName
-	}()
-
-	tests := []struct {
-		name         string
-		expected     []string
-		notExpected  []string
-		exportConfig ExportConfig
-	}{
-		{
-			name: "exportConfig is completely empty",
-			exportConfig: ExportConfig{
-				ChartsPath:      "",
-				OsType:          Linux,
-				RancherVersion:  "",
-				GithubEndpoints: []GithubEndpoint{},
-			},
-			expected:    []string{},
-			notExpected: []string{"rancher/ui-plugin-catalog"},
-		},
-		{
-			name: "only extensions is set in exportConfig",
-			exportConfig: ExportConfig{
-				ChartsPath:     "",
-				OsType:         Linux,
-				RancherVersion: "",
-				GithubEndpoints: []GithubEndpoint{
-					{URL: server.URL},
-				},
-			},
-			expected:    []string{"rancher/ui-plugin-catalog"},
-			notExpected: []string{},
-		},
-	}
-
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			imagesList, _, err := GetImages(tt.exportConfig, make(map[string][]string), []string{}, map[string]rketypes.RKESystemImages{})
```

Comment:
We aren't removing `GetImages` completely, just the RKESystemImages so this test is still required unless I am missing something?